src/tsickle.ts

```diff
@@ -630,6 +638,21 @@ class Annotator extends ClosureRewriter {
     return toArray(reexports.keys());
   }
 
+  private writeModuleSpecifier(moduleSpecifier: ts.Expression) {
+    let moduleId = (moduleSpecifier as any).text;
```

Comment:
https://github.com/angular/tsickle/blob/d3bb6897edddee5df0dcb5b38fbb57b77393d436/src/tsickle.ts#L99 is an example